src/Tooltip/TooltipDictionary.yaml

```
@@ -6439,6 +6439,305 @@ Bitrise_stepName_StepConfig:
 gitops-agent-disaster-recovery-create:
     content: The Agent Name has been overwritten to the Primary Agent Selected & the Import operation for the Secondary Agent is disabled. Please see additional information [here](https://docs.harness.io/article/52r3l5q67u).
     width: "400"
+tune_fault_total_chaos_duration:
```

```suggestion
chaos_tune_fault_total_chaos_duration:
```

same for all `tune_fault_{tunables}`